back/app/serializers/web_api/v1/custom_field_serializer.rb

```diff
@@ -2,7 +2,7 @@
 
 class WebApi::V1::CustomFieldSerializer < WebApi::V1::BaseSerializer
   attributes :key, :input_type, :title_multiloc, :required, :ordering,
-    :enabled, :code, :created_at, :updated_at, :logic
+    :enabled, :code, :created_at, :updated_at, :logic, :random_option_ordering
```

Comment:
I think this should be implemented as a relationship, if possible (for consistency with our API specs).

Reply:
Sorry I got confused 😅 You can ignore my comment.